src/extensions/http_api/definitions.js

```diff
@@ -125,11 +125,18 @@ exports.install = ({ baseUrl = '' } = {}) => {
         this.httpApiClient.setQuery(Cast.object(this.state.populateObject(step.rowsHash())))
     })
 
-    Given(/^(?:I )?pick response json (.+) as (.+)$/, function (path, key) {
+    /**
+     * Pick a value from previous json response and set it to state
+     */
+    Given(/^(?:I )?pick response json (body|headers) (.+) as (.+)$/, function (
```

Comment:
That would be a breaking change, we could make (body|headers) with space optional (something like `(?:(body|headers) )?` maybe? I'll let you test it), and assume body by default

Reply:
By the way, wouldn't `header` instead of `headers` be more natural ? "I pick header foo" meaning that I want the header named "foo".

WDYT @tienduy-nguyen ?

Reply:
That's exactly what I want, but I couldn't know how to use the regex like you propos here. Done!